src/main/java/org/team1540/robot2019/commands/hatch/PlaceHatch.java

```diff
@@ -2,11 +2,16 @@
 
 import edu.wpi.first.wpilibj.command.CommandGroup;
 import edu.wpi.first.wpilibj.command.WaitCommand;
+import org.team1540.robot2019.Robot;
 import org.team1540.robot2019.Tuning;
 
 public class PlaceHatch extends CommandGroup {
 
     public PlaceHatch() {
+        if (Robot.hatch.retracted()) {
```

Comment:
This doesn't only do the extend+wait when the hatch is retracted—the check is performed on robot boot. Probably want a `SimpleConditionalCommand`.